templates/sleep/sleep.jinja2

```diff
@@ -13,9 +13,9 @@
           - functional
         run:
           steps:
-            - dmesg -n 6  # Silence KERN_INFO as it confuses LAVA on wakeup
             - for i in $(seq 1 10); do lava-test-case rtcwake-mem-$i --shell rtcwake -d rtc0 -m mem -s 1; done
             - for i in $(seq 1 10); do lava-test-case rtcwake-freeze-$i --shell rtcwake -d rtc0 -m freeze -s 1; done
+      lava-signal: kmsg
```

Comment:
This isn't merged yet, and for various reasons doesn't actually work. Maybe better to leave the dmesg call?

Reply:
Yes it's merged: https://review.linaro.org/#/c/25377/
and applied in `lab-collabora`.  As we control in which labs which test plans are run, this should be fine.

Reply:
Hah, I leave LAVA for a week... :)
Ok cool, I'll merge this